core/src/main/scala/scalaz/syntax/std/BooleanOps.scala

```diff
@@ -282,7 +282,10 @@ final class BooleanOps2(self: Boolean) {
    * Returns the first argument in `\/-` if this is `true`, otherwise the second argument in
    * `-\/`.
    */
-  final def either[A](a: => A): ConditionalEither[A] = new ConditionalEither(a)
+  final def toEither[A](a: => A): ConditionalEither[A] = new ConditionalEither(a)
+
+  @deprecated("Use `toEither`", "7.3.0")
+  final def either[A](a: => A): ConditionalEither[A] = toEither(a)
```

Comment:
This one is a bit special. It's used as `bool.either(right).or(left)` so I don't think it should be renamed.

Reply:
nice catch @joroKr21 !